pkgs/os-specific/linux/audit/default.nix

```
@@ -33,25 +39,29 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  # TODO: Remove the musl patches when
-  #         https://github.com/linux-audit/audit-userspace/pull/25
-  #       is available with the next release.
-  patches = stdenv.lib.optional stdenv.hostPlatform.isMusl [
-    (
-      let patch = fetchpatch {
-            url = "https://github.com/linux-audit/audit-userspace/commit/d579a08bb1cde71f939c13ac6b2261052ae9f77e.patch";
-            name = "Add-substitue-functions-for-strndupa-rawmemchr.patch";
-            sha256 = "015bvzflg1s1k5viap30nznlpjj44a66khyc8yq0waa68qwvdlsd";
-          };
-      in
-        runCommand "Add-substitue-functions-for-strndupa-rawmemchr.patch-fix-copyright-merge-conflict" {} ''
+  patches =
+    # This patch allows us to use binary/directory permissions 0555 or 0550,
+    # which are the default ones for the nix store.
+    [ ./allowed-permissions.patch ]
+    # TODO: Remove the musl patches when
+    #         https://github.com/linux-audit/audit-userspace/pull/25
+    #       is available with the next release.
+    ++ stdenv.lib.optional stdenv.hostPlatform.isMusl [
+      (
+        let patch = fetchpatch {
+          url = "https://github.com/linux-audit/audit-userspace/commit/d579a08bb1cde71f939c13ac6b2261052ae9f77e.patch";
+          name = "Add-substitue-functions-for-strndupa-rawmemchr.patch";
+          sha256 = "015bvzflg1s1k5viap30nznlpjj44a66khyc8yq0waa68qwvdlsd";
+        };
+        in
+        runCommand "Add-substitue-functions-for-strndupa-rawmemchr.patch-fix-copyright-merge-conflict" { } ''
```

I would suggest something closer to to minimize the current and future diffs:

```suggestion
  patches = [
    # This patch allows us to use binary/directory permissions 0555 or 0550,
    # which are the default ones for the nix store.
    ./allowed-permissions.patch
  ] ++ stdenv.lib.optional stdenv.hostPlatform.isMusl [
    # TODO: Remove the musl patches when
    #         https://github.com/linux-audit/audit-userspace/pull/25
    #       is available with the next release.
    (
      let patch = fetchpatch {
        url = "https://github.com/linux-audit/audit-userspace/commit/d579a08bb1cde71f939c13ac6b2261052ae9f77e.patch";
        name = "Add-substitue-functions-for-strndupa-rawmemchr.patch";
        sha256 = "015bvzflg1s1k5viap30nznlpjj44a66khyc8yq0waa68qwvdlsd";
      };
      in
      runCommand "Add-substitue-functions-for-strndupa-rawmemchr.patch-fix-copyright-merge-conflict" { } ''
```

---

Yep, fixed as suggested :+1: